src/entities/trade.test.ts

```diff
@@ -1160,6 +1192,50 @@ describe('Trade', () => {
   })
   // v3 sdk price impact tests
   describe('#priceImpact', () => {
```

Comment:
Should we add non-FOT cases to ensure we don't have any regressions?

Reply:
oh there are already non FOT cases from before, lines 1238 - 1298, do u think we should add more tho

Reply:
Ah missed that, all good!